packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -808,6 +747,9 @@ class EditableText extends StatefulWidget {
     this.spellCheckConfiguration,
     this.magnifierConfiguration = TextMagnifierConfiguration.disabled,
     this.undoController,
+    this.getSemanticsCanCopy = defaultGetSemanticsCanCopy,
```

Comment:
Good catch, I meant to explain this but hadn't gotten around to it. See the updated description of this PR and my addition to the docs about this (https://github.com/flutter/flutter/pull/141310/commits/537ede3ad914e5da546bf00af3204b67464667dc) and let me know what you think.

They are used in editable_text_test.dart. I needed them to fix some tests, so I think they're needed in this PR, unless we were to scrap this functionality.

Reply:
Got it, thanks.

Wondering if this could be simplified to a single API getter EditableText.toolbarSemanticsActions which by default would return something like:
```
  List<SemanticsAction> get toolbarSemanticsActions => <SemanticsAction>[
    if (cutEnabled) SemanticsAction.cut,
    if (copyEnabled) SemanticsAction.copy,
    if (pasteEnabled) SemanticsAction.paste,
  ];
```

Reply:
Yeah that seems to be a cleaner API. I've now done it using a custom record instead of a List (https://github.com/flutter/flutter/pull/141310/commits/4c10f4c0385ed64d40b148fc7150d03f899ef1c0), to avoid confusion about all of the existing [SemanticsAction](https://main-api.flutter.dev/flutter/dart-ui/SemanticsAction-class.html)s.